<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>565</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_NewFile"/>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="action_OpenRO"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_Help"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Язык</string>
    </property>
    <addaction name="action_English"/>
    <addaction name="action_Russian"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Тема</string>
     </property>
     <addaction name="action_LightTheme"/>
     <addaction name="action_DarkTheme"/>
    </widget>
    <addaction name="action_changeShortcuts"/>
    <addaction name="menu_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_NewFile">
   <property name="text">
    <string>Новый файл</string>
   </property>
   <property name="toolTip">
    <string>Создать новый файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Сохранить...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_OpenRO">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Открыть только для чтения...</string>
   </property>
  </action>
  <action name="action_English">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="action_Russian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string notr="true">Русский</string>
   </property>
  </action>
  <action name="action_changeShortcuts">
   <property name="text">
    <string>Изменить сочетания клавиш</string>
   </property>
  </action>
  <action name="action_LightTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Светлая</string>
   </property>
  </action>
  <action name="action_DarkTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Темная</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
